x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -118,9 +112,14 @@ export function WorkspacePanelWrapper({
     warningMessages.push(...requestWarnings);
   }
   return (
-    <EuiPageTemplate direction="row" grow={true} offset={0} minHeight={0} responsive={[]}>
+    <EuiPageTemplate direction="column" grow={false} offset={0} minHeight={0}>
```

Comment:
This is a bit off-topic, but it's probably worth having a future conversation on this placement of `EuiPageTemplate`. I know we had it here previously and we're just attempting to fix the scrolling issue with the visualization legend, but it feels wrong to me that the `main` element packaged within the `EuiPageTemplate` component doesn't encompass the full main content of the page (i.e. field list, workspace, suggestions, layers, etc.). Probably a conversation for a different time and issue, but wanted to note it nonetheless.